tests/manual/addonStore.md

```diff
@@ -100,38 +100,40 @@ You can do this by:
 
 ## Updating add-ons
 
-### Updating from add-on installed from add-on store
-1. [Install an add-on from the add-on store](#install-add-on)
+### Simulate creating an updatable add-on from the Add-on Store
+Without having a previously installed add-on get an update, it is hard to naturally test updatable add-ons.
+This process allows you to spoof an update for an add-on.
+
+1. [Install an add-on from the Add-on Store](#install-add-on)
 For example: "Clock".
 1. Go to your NVDA user configuration folder:
-    - For source: `.\source\userConfig`
-    - For installed copies: `%APPDATA%\nvda`
+    - For source: `.\source\userConfig\addons`
+    - For installed copies: `%APPDATA%\nvda\addons`
 1. To spoof an old release, we need to edit 2 files:
-    - Add-on store JSON metadata
-        - Located in: `.\addonStore\addons\`.
-        - Example: `source\userConfig\addonStore\addons\clock.json`
+    - Add-on Store JSON metadata
+        - Example: `source\userConfig\addons\clock.json`
         - Edit "addonVersionNumber" and "addonVersionName": decrease the major release value number.
     - Add-on manifest
-        - Located in: `.\addons\`.
         - Example: `source\userConfig\addons\clock\manifest.ini`
         - Edit "version": decrease the major release value number to match earlier edits.
-1. Open the add-on store
-1. Ensure the same add-on you edited is available on the add-on store with the status "update".
+
+### Updating from add-on installed from Add-on Store

```

Comment:
In addition to the wording clarity suggestion below, I would like to echo the AI's opinion that we should have blank lines around headings. It makes reading easier, and is far more standard in non-NVA MD documents.

I would think it also made reading the source visually easier, but in reading it with speech, reading rapidly and hitting blank lines around headings helps them stand out in a large wall of speech.

```suggestion
### Updating from add-on originally installed via Add-on Store
```